docs/UserGuide.md

```diff
@@ -205,8 +233,19 @@ Format: `addscore -a <assessment_name> (-n <student_name> | -i <student_id>) -s
 * If multiple students have the same name, NUSNET ID needs to be used to identify them.
 
 Examples:
-* `addscore -a Midterm -n Van Nhi -s 95` records score for `Van Nhi` in `Midterm` to be `95` percent.
-* `addscore -a P01 -i E0123456 -s 75.25` records score for NUSNET ID `E0123456` in `P01` to be `75.25` percent.
+* `addscore -a Midterm -n Van Nhi -s 95` 
+  * records score for `Van Nhi` in `Midterm` to be `95` percent.
```

Comment:
Hm should this be `95.00` since we record `95.00` into the system instead of just `95`? Just my opinion what do you think?

```suggestion
  * records score for `Van Nhi` in `Midterm` to be `95.00` percent.
```